CMakeLists.txt

```diff
@@ -0,0 +1,119 @@
+# Copyright 2022
```

Comment:
You should probably add your name here. (I think that's an annoyance here in the library. It would be good to switch to "libsecp256k1 contributors" but last time I brought that up, it wasn't clear if this is legally okay.)

Reply:
[Updated](https://github.com/bitcoin-core/secp256k1/pull/1113#issuecomment-1171325257).